src/server/sams/api/produce/assets.py

```diff
@@ -119,6 +119,42 @@ def unlock_asset(asset_id: str):
     return service.patch(ObjectId(asset_id), updates)
 
 
+@assets_produce_bp.route('/produce/assets/user/<Ids>', methods=['PATCH'])
+def unlock_asset_by_user(Ids: list):
+    """
+    Uses user_id, session_id and unlocks the corresponding assets
+    """
+    Ids = Ids.split(',')
```

Comment:
This can be restrictive as to what characters may be in the user/session IDs (also the url doesn't indicate an action to unlock Asset(s)).

It might be better to use:
* `/produce/assets/unlock_user_session` as the URL
* `request.json.get` to retrieve these IDs from the body

This way the URL is descriptive of the action and the IDs can be any string with less restrictions.